portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body class="MainContainer">
+  <body class="MainContainer" onload="showComments()">
```

Comment:
Instead of adding onload to body container, you can add a handler to window.onload in JS.

```
window.onload(() => {
  // Do something.
});
```